code/lib/builder-vite/jest.config.js

```diff
@@ -4,4 +4,5 @@ const baseConfig = require('../../jest.config.node');
 module.exports = {
   ...baseConfig,
   displayName: __dirname.split(path.sep).slice(-2).join(path.posix.sep),
+  transformIgnorePatterns: ['node_modules/(?!rehype-slug)/'],
```

Comment:
instead of adding this here, you should be able to add the module to a global list of transformed modules we have here:

https://github.com/storybookjs/storybook/blob/next/code/jest.config.base.js#L22-L47

Reply:
@JReinhold I added it in the jest.config.base.js but the unit test again started failing, is there something I might be doing wrong?